test/test_helper.exs

```diff
@@ -1 +1,5 @@
+# Configure access key and secret key with dummy value to make AuthCache work.
+System.put_env("AWS_ACCESS_KEY_ID", "dummy")
+System.put_env("AWS_SECRET_ACCESS_KEY", "dummy")
```

Comment:
It would be better to set these in `test.exs` directly.

Reply:
Do you mean `config/test.exs`?  If so, agreed.

Reply:
I do yes.

Reply:
Fixed in 6e7307b